dev-tools/v2tool/v2-config.yml

```diff
@@ -0,0 +1,26 @@
+args:
+  - loadgenerator
+rules:
+  input/loadgenerator:
+    start:
+      OnStart: {}
+    stop:
+      After:
+        time: 10s
+inputs:
+  - id: input/loadgenerator
+    name: test-integration-1
+    type: input/loadgenerator
+    start:
+      OnStart
+    stop:
+      After:
+        time: 10s
```

Comment:
I'm just a bit puzzled why this is here and also almost repeated on L5-L9. If they don't match, which one prevails?

Reply:
Ah, my bad. The one in the `rules` section is the functional one.